src/main.rs

```diff
@@ -100,6 +100,15 @@ fn main() {
                                .number_of_values(1)
                                .takes_value(true))
 
+                          .arg(Arg::with_name("keep_dir")
+                               .help("Keep only files/directories specified as globs")
+                               .long("keep-only")
+                               .value_name("PATH")
+                               .multiple(true)
+                               .number_of_values(1)
```

Comment:
I don't think you want that (I fixed it for ignore: https://github.com/mozilla/grcov/pull/459)

Reply:
Please forget my comment, it's probably better like this.